### PR TITLE
test(dm): fix unexpected sync-diff temporarily

### PR DESCRIPTION
**Pull request by okJiang**
### What problem does this PR solve? 

https://github.com/pingcap/tiflow/issues/4029

### What is changed and how it works?

- add comment to sync-diff probably wrong.

### Check List 

Tests 

 - Integration test

### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

File: dm/tests/shardddl4/run.sh [hunk 1]
```diff
@@ -31,7 +31,7 @@ function DM_119_CASE {
 	#         "query-status test" \
 	#         "because schema conflict detected" 1
 	# fi
-	check_sync_diff $WORK_DIR $cur/conf/diff_config.toml 10 "fail"
+	# check_sync_diff $WORK_DIR $cur/conf/diff_config.toml 10 "fail"
```
Comment (lonng) [inline dm/tests/shardddl4/run.sh:34]: Prefer to add a TODO comment here.

Comment (okJiang) [reply dm/tests/shardddl4/run.sh:36]: add it in https://github.com/pingcap/tiflow/pull/4031/commits/5458724ae9b9fa747794ff2d0ce4524ad94c2107. 

ptal

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Review by lonng — APPROVED** [review]

**Pull request merged**